test/parallel/test-repl-import-referrer.js

```diff
@@ -24,4 +24,6 @@ child.on('exit', common.mustCall(() => {
 
 child.stdin.write('await import(\'./message.mjs\');\n');
 child.stdin.write('.exit');
-child.stdin.end();
+setTimeout(() => {
+  child.stdin.end();
+}, 100);
```

Comment:
```suggestion
}, common.platformTimeout(100));
```

Reply:
sounds good to me, but just so that I know, can you explain what the difference is? 🙂

Reply:
Sure, the reason for using `platformTimeout` is because timer's performance varies between platforms in our CI. E.g. on `AIX` and `IBMi` we increase the timeout by `2` and on `Riscv64` we even need to increase by `4` to reduce the platform related flakness.

Reply:
ah ok I see, thank you very much for the explanation! :heart_hands: :pray: